test/ex_webrtc/browser_sdp_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule ExWebRTC.BrowserSDPTest do
+  use ExUnit.Case, async: true
+
+  alias ExWebRTC.{MediaStreamTrack, PeerConnection, SessionDescription}
+
+  for browser <- ["firefox"] do
+    test "#{browser} SDP offer" do
```

Comment:
Just to make sure we can successfuly parse remote SDP offers both from chromium and ff. Thanks to this I also spotted a bug with DTLS cert fingerprint